src/pages/index.js

```diff
@@ -24,7 +40,415 @@ const profileModalTitleInput =
+      });
+  }
+);
+avatarUpdateModal.setEventListeners();
+
+//
+// ModalConfirmDelete Instantiation
+//
+const deleteConfirmation = new ModalConfirmDelete("#delete-card-confirm-modal");
+deleteConfirmation.setEventListeners();
+
+//
+// Form Validation
+//
+const formValidators = {};
+
+const enableValidation = (config) => {
+  const formList = Array.from(document.querySelectorAll(".modal__form"));
+
+  formList.forEach((formElement) => {
+    const validator = new FormValidator(config, formElement);
+
+    const formName = formElement.getAttribute("name");
+
+    formValidators[formName] = validator;
+    validator.enableValidation();
+  });
+};
+
+enableValidation(configItems);
+
+//
+//Event Handlers
+//
+function handleProfileModalSubmit(data) {
+  api
+    .updateProfileInfo(data.title, data.description)
+    .then((userData) => {
+      userInfo.setUserInfo({
+        title: userData.name,
+        description: userData.about,
+      });
+      profileModal.close();
+    })
+    .catch((err) => console.error(`Failed to update users info: ${err}`))
+    .finally(() => {
+      profileModal.setLoading(false);
+    });
+}
+
+function handleAddCardModalSubmit(data) {
+  api
+    .createNewCard({ name: data.title, link: data.url })
+    .then((cardData) => {
+      const cardElement = createCard(cardData);
+      section.addItem(cardElement);
+
+      addCardModalForm.reset();
+
+      addCardModal.close();
+    })
+    .catch((err) => console.error(`Failed to add card: ${err}`))
+    .finally(() => {
+      addCardModal.setLoading(false);
+    });
+}
+/* Previous handleAddCardModalSubmit
+   function handleAddCardModalSubmit(data) {
+  api
+    .createNewCard({ name: data.title, link: data.url })
+    .then((cardData) => {
+      const cardElement = createCard(cardData);
+      section.addItem(cardElement);
+
+      addCardModalForm.reset();
+      formValidators["add-card-form"].disableSubmitButton();
+      addCardModal.close();
+    })
+    .catch((err) => console.log(`Failed to add card: ${err}`));
+}
+*/
+
+const handleImageClick = (name, link) => {
+  imagePreviewModal.open({ name, link });
+};
+
+//
+//Delete a Card using api.deleteCard method
+//
+function handleDeleteCard(card) {
+  deleteConfirmation.open();
+  deleteConfirmation.setDeleteConfirmation(() => {
+    return api
+      .deleteCard(card.getCardId())
+      .then(() => {
+        deleteConfirmation.close();
+        card.removeCard();
+      })
+      .catch(console.error);
+  });
+}
+/* Previouse handleDeleteCard
+function handleDeleteCard(cardId) {
+  deleteConfirmation.open();
+  deleteConfirmation.setDeleteConfirmation(() => {
+    api
+      .deleteCard(cardId._id)
+      .then(() => {
+        deleteConfirmation.close();
+        cardId.handleDeleteButton();
+      })
+      .catch(console.error);
+  });
+}
+*/
+
+//
+//Like a Card using api.likeCard and api.dislikeCard methods
+//
+function handleLikeCard(card) {
+  if (card._isLiked) {
```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.